can/uwrt_mars_rover_can.yaml

```diff
@@ -0,0 +1,1855 @@
+uwrt_mars_rover_can_bus:
+  bus_frequency: 1000000
+  canid_filter_mask: 2016
+  canid_filters:
+    0: NO_FILTER
```

Comment:
CAN IDs are represented in decimal in the yaml. May be worth looking into how to have hex representation instead